.github/workflows/dashboard_deploy_template.yaml

```diff
@@ -0,0 +1,80 @@
+name: Dashboard Deploy
+
+on:
+  workflow_call:
+    inputs:
+      parameter_file:
+        description: "Path to the parameters file"
+        required: true
+        type: string
+
+jobs:
+  deployShiny:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install yq
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y yq
+
+      - name: Read parameters
+        run: |
+          DASHBOARD_NAME=$(yq '.dashboard_name' ${{ inputs.parameter_file }})
+          DEPLOY_TARGET=$(yq '.deploy_target' ${{ inputs.parameter_file }})
+          INSTALL_GDAL=$(yq '.install_gdal' ${{ inputs.parameter_file }})
+          EXTRA_PACKAGES=$(yq '.extra_packages | join(" ")' ${{ inputs.parameter_file }})
+
+          echo "DASHBOARD_NAME=$DASHBOARD_NAME" >> $GITHUB_ENV
+          echo "DEPLOY_TARGET=$DEPLOY_TARGET" >> $GITHUB_ENV
+          echo "INSTALL_GDAL=$INSTALL_GDAL" >> $GITHUB_ENV
+          echo "EXTRA_PACKAGES=$EXTRA_PACKAGES" >> $GITHUB_ENV
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install GDAL
```

Comment:
I think it would be clearer if the name of this step was "Install terra dependencies" as that highlights the name of the R package this relates to?

Reply:
Thanks for the review @cjrace. That makes sense, I have updated the step name, variables and parameters accordingly.